### PR TITLE
add Fall 2020 officers

### PR DESCRIPTION
**Pull request by cg505**
Also, update the Officers page to support historical committee heads.

### CODE REVIEW

File: ocfweb/docs/templates/docs/officers.html [hunk 1]
```diff
@@ -142,6 +143,24 @@ <h5>Deputies:</h5>
...
+          {% if term.heads %}
+            <tr>
+              <td style="border: none;" />
+              <td colspan="2">
+                <details>
+                  <summary style="display: list-item;">
+                    Committee Heads
+                  </summary>
+                  {% for committee in term.heads %}
+                    <h5>{{ committee.name }} Heads</h5>
```
Comment (dkess) [inline ocfweb/docs/templates/docs/officers.html:155]: Probably don't need the word "Heads" on this line, the word head is used a lot

Comment (cg505) [reply ocfweb/docs/templates/docs/officers.html:155]: I am worried if we do not put "Heads" here then people will think the heads are the whole committee.

Comment (dkess) [reply ocfweb/docs/templates/docs/officers.html:155]: The word is still used in the dropdrown menu so I don't think it'll have that effect

**Comment by cg505** [comment]
sorry im so slow

File: ocfweb/docs/templates/docs/officers.html (hunk 1 above)
Comment (cg505) [reply ocfweb/docs/templates/docs/officers.html]: ok, done

**Pull request merged**